core/src/test/resources/newlineBeforeLambdaParams/LambdaNewline.stat

```diff
@@ -0,0 +1,12 @@
+40 columns                              |
+<<< newline before lambda params {
+lst.map { x =>
+      println(x)
+  x + 1
+}
+>>>
+lst.map {
+  x =>
+    println(x)
+    x + 1
+}
```

Comment:
What happens for this?

```
lst.map { x => x }
```

Reply:
hey, I think its an overkill to open an issue for this

but how can I avoid

```
def f(x) = {
    x => x
}
```

becoming

```
def f(x) = {
    x =>
      x
}
```

?